api/comments/views.py

```diff
@@ -37,8 +34,8 @@ class CommentMixin(object):
     def get_comment(self, check_permissions=True):
         pk = self.kwargs[self.comment_lookup_url_kwarg]
         try:
-            comment = Comment.find_one(Q('guids___id', 'eq', pk) & Q('root_target', 'ne', None))
-        except NoResultsFound:
+            comment = Comment.objects.filter(guids___id=pk).exclude(root_target=None).get()
```

Comment:
Django will actually handle `None` for you.

```
In [170]: Node.objects.filter(preprint_file=None).count()
Out[170]: 2687

In [171]: Node.objects.filter(preprint_file__isnull=True).count()
Out[171]: 2687
```

Although I prefer `root_target__isnull=True` for consistency, so I'll change this.